pages/static-security-analysis.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Static Security Analysis
+permalink: /static-security-analysis/
+---
+
+Static analysis is an important part of the development process, and is required for ATO. There are two main types of static security testing that needs to be done: dependency analysis, where the ruby gems, python modules, and js packages your app uses are checked against a list of known vulnerabilities; and code analysis in which your code is checked against a list of antipatterns. There are tools for JS, Ruby, and Python, and you are encouraged to set up this scanning early on in the development cycle to prevent unexpected delays when it's time to get your ATO.
+
+## Javascript
+
+### Dependency analysis
+
+We recommend using the Node Security Project's [NSP tool](https://nodesecurity.io/tools) to scan your package.json or npm-shrinkwrap.json file for known security issues.
+
+To install:
+
+    $ npm install nsp
+
+To run, from your project directory:
+
+    $ nsp check
+
+### Static code analysis
+
+There are several tools available for running analysis on JS projects, but the most full featured seems to be [ESLint](https://eslint.org). ESLint doesn't offer security scanning out of the box, but it is pluggable and Mozilla has provided a set of rules that mimic the behavior of their now deprecated ScanJS. These rules are available on [github](https://github.com/mozfreddyb/eslint-plugin-scanjs-rules).
+
+To install:
+
+    $ npm install eslint
+    $ npm install eslint-plugin-scanjs-rules
+
+___TODO: PROVIDE BASELINE .eslintrc___
```

Comment:
I'm going to channel @jessieay here and ask, "can we move these inline TODOs to issues, so that we can @mention people, assign them, have conversations around them, etc.?"